core/amber/src/test/scala/edu/uci/ics/texera/workflow/operators/projection/ProjectionOpDescSpec.scala

```diff
@@ -58,7 +69,12 @@ class ProjectionOpDescSpec extends AnyFlatSpec with BeforeAndAfter {
   }
 
   it should "raise IllegalArgumentException with multiple input source Schema" in {
-    projectionOpDesc.attributes ++= List("field1", "field2")
+
+    projectionOpDesc.attributes ++= List(
+      new AttributeUnit("field2", "f2"),
+      new AttributeUnit("field1", "f1")
+    )
+
     assertThrows[IllegalArgumentException] {
       projectionOpDesc.getOutputSchema(Array(schema, schema))
     }
```

Comment:
you should add test cases to test the functionality. e.g., if user enters a duplicate alias, if a user doesn't specify an alias ...etc